src/c_kzg_4844.c

```diff
@@ -1755,37 +1672,49 @@ C_KZG_RET load_trusted_setup(
     const uint8_t *g2_bytes,
     size_t n2
 ) {
-    uint64_t i;
-    blst_p2_affine g2_affine;
-    g1_t *g1_projective = NULL;
     C_KZG_RET ret;
 
-    out->fs = NULL;
+    out->max_width = 0;
+    out->roots_of_unity = NULL;
     out->g1_values = NULL;
     out->g2_values = NULL;
 
     /* Sanity check in case this is called directly */
     CHECK(n1 == TRUSTED_SETUP_NUM_G1_POINTS);
     CHECK(n2 == TRUSTED_SETUP_NUM_G2_POINTS);
 
+    /* It's the smallest power of 2 >= n1 */
```

Comment:
```suggestion
    /* 1<<max_scale is the smallest power of 2 >= n1 */
```
The previous wording sounded like max_scale (rather than 1<<max_scale) was a power of 2.